app/assets/javascripts/discourse/app/lib/ember-action-modifier.js

```diff
@@ -16,7 +16,12 @@ export const actionModifier = modifier(
     { on, bubbles, preventDefault, allowedKeys }
   ) => {
     const handler = (event) => {
-      const fn = typeof callback === "string" ? context[callback] : callback;
+      let fn;
+      if (typeof callback === "string") {
+        fn = context[callback] ?? context.actions?.[callback];
```

Comment:
It's possible for both `context[callback]` and `context.actions[callback]` to exist - do we know which way around Ember itself prioritises them? My gut says it's probably `actions[callback]` first, but I'm not sure 🤔

Reply:
Your gut is right :)

I'll add tests to ensure that the correct precedence is used.

Reply:
@davidtaylorhq ok, updated with precedence fix. Good catch!